app/models/stats/claim_stats.rb

```diff
@@ -0,0 +1,11 @@
+module Stats
+  class ClaimStats < Claim
+    scope :started, -> { where.not(password_digest: nil, state: 'submitted').within_timeframe }
+    scope :started_count, -> { started.count }
```

Comment:
This feels like a bit of a subversion of `scope` - it works because `scope` is essentially the same as defining a class method, but I think it's a bit confusing in terms of intention: it returns something that can't be chained (if I've understood correctly).